tests/test_decorators.py

```diff
@@ -33,35 +36,35 @@ def setUp(self):
 
     def test_200_ok_if_no_token_set(self):
         watchman_settings.WATCHMAN_TOKEN = None
-        response = self.client.get(reverse('watchman.views.status'))
+        response = self.client.get(reverse('status'))
```

Comment:
Yes, reversing dotted module paths has been deprecated in 1.8 and was removed in 1.10.